packages/cli/src/commands/setup/auth/templates/auth.ts.template

```diff
@@ -108,7 +108,7 @@ export const requireAuth = ({ roles }: { roles: AllowedRoles }) => {
     throw new AuthenticationError("You don't have permission to do that.")
   }
 
```

Comment:
Remind me again what this template is used for, I'm a little confuzzled. I thought each provider had their own auth template?

Reply:
This is a default auth that ships so that the auth directives have something to invoke for a dummy @requireAuth